docs/api.md

```diff
@@ -13,6 +15,40 @@
 <dd></dd>
 </dl>
 
+<a name="Application"></a>
+
+## Application
```

Comment:
Did you type this out manually? I think that this has been auto generated somehow. @nguyer ?